universal-application-tool-0.0.1/app/controllers/admin/AdminProgramController.java

```diff
@@ -54,17 +54,20 @@ public AdminProgramController(
     this.formFactory = checkNotNull(formFactory);
   }
 
+  /** Return a HTML page displaying all programs of current live version. */
```

Comment:
suggestion (blocking):
```suggestion
  /** Return a HTML page displaying all programs of the current live version and all programs of the current draft version. */
```

Reply:
Done.